src/fw_algorithms.jl

```diff
@@ -605,7 +605,9 @@ function stochastic_frank_wolfe(
             callback(state)
         end
 
-        @memory_mode(memory_mode, x = (1 - gamma) * x + gamma * v)
+        d = similar(x)
```

Comment:
`d` should not be initialized before starting the loop

Reply:
The loop starts in in l.527 and ends in l.633, so d is initialized in the loop. Or are you referring to something else?